app/controllers/companies_controller.rb

```diff
@@ -10,13 +10,15 @@ def show
   def create
     company = Company.new(company_params)
     if company.save
-      User.create(email: company_params[:email],
-                  name: company_params[:name],
-                  title: company_params[:title],
-                  password: User.generate_password,
-                  company_id: company.id
-                 )
+      password = User.generate_password
+      user = User.create(email: company_params[:email],
+                          name: company_params[:name],
+                          title: company_params[:title],
+                          password: password,
+                          company_id: company.id
+                          )
       flash[:success] = "Welcome #{company.organization}"
+      UserEmailer.send_welcome_email(user, password).deliver_now
```

Comment:
I believe that is correct, this is from the ActionMailer docs 

"Notifier.welcome(User.first).deliver_now   # sends the email"
"Notifier.welcome(User.first).deliver_later # enqueue email delivery as a job through Active Job"